main.go

```diff
@@ -40,6 +44,30 @@ func main() {
 		log.Fatal("Start MySQL Server!")
 	}
 
+	// Read redis connection/auth info out of REDIS_URL env var already defined
+	u, err := url.Parse(os.Getenv("REDIS_URL"))
+	if err != nil {
+		log.Fatal("No REDIS_URL env var?")
```

Comment:
Again, if caching is not working we don't want to crash the application, but we should definitely warn the user.